R/pkg/R/DataFrame.R

```diff
@@ -1384,7 +1384,7 @@ setMethod("saveAsTable",
                                     "org.apache.spark.sql.parquet")
             }
             allModes <- c("append", "overwrite", "error", "ignore")
-            if (!(mode %in% allModes)) {
+            if (! (mode %in% allModes)) {
```

Comment:
In this and in the one above I think we can remove the parens ? i.e. `! mode %in% allModes` might look better ? Any thoughts on this ?